src/picknik_ur_site_config/objectives/getdoorgraspableobjectsubtree.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="GetDoorGraspableObjectSubtree">
+  <!-- ////////// -->
+  <BehaviorTree ID="GetDoorGraspableObjectSubtree">
+    <Control ID="Sequence" name="TopLevelSequence">
+      <!-- Load the objectives needed for this behavior -->
+      <!-- Update the admittance controller and reset the FTS -->
```

Comment:
Just as a heads-up, admittance control does not work in Gazebo so it has to be overridden somehow for that.